src/openzaak/utils/models.py

```diff
@@ -12,3 +22,100 @@ def clone_object(instance):
     except AttributeError:
         pass
     return cloned
+
+
+class ImportStatusChoices(models.TextChoices):
+    pending = "pending", _("Openstaand")
+    active = "active", _("Actief")
+    finished = "finished", _("Voltooid")
+    error = "error", _("Onderbroken")
+
+    @classproperty
+    def visible_choices(cls):
+        return {cls.pending, cls.active, cls.finished, cls.error}
+
+    @classproperty
+    def started_choices(cls):
+        return {cls.pending, cls.active}
+
+    @classproperty
+    def report_choices(cls):
+        return {cls.finished, cls.error}
+
+
+class ImportTypeChoices(models.TextChoices):
+    documents = "documenten", _("Enkelvoudige informatie objecten")
+
+
+class Import(models.Model):
```

Comment:
I wasn't really sure in which app we would like to add this model hence its in `utils` at the moment

Reply:
Let's put it into separate app. I think it deserves it

Reply:
On the second thought maybe it could be put into `components.documenten.import` app, since we don't make it generic

Reply:
I created the `import_data` app for the import code. `components.documenten.import` would probably clash with the current  `openzaak.components.documenten` app